### PR TITLE
HPCC-28331 ES Translations for 8.10.X

### PR DESCRIPTION
**Pull request by kunalaswani**
Translations added for Spanish (ES).

Signed-off-by: Kunal Aswani <kunal.aswani@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: esp/src/src/nls/es/hpcc.ts [hunk 1]
```diff
@@ -338,8 +343,15 @@ export = {
...
     Generate: "Generar",
+    GetDFSCSV: "CSV de DFS",
+    GetDFSMap: "Mapa DFS",
+    GetDFSParents: "Padres de DFS",
+    GetLogicalFile: "Archivo lógico",
+    GetLogicalFilePart: "Parte del archivo lógico",
     GetPart: "Obtener parte",
+    GetProtectedList: "Lista Protegida",
     GetSoftwareInformation: "Obtener informacion del software",
+    GetValue: "Valor",
```
Comment (rpastrana) [inline esp/src/src/nls/es/hpcc.ts:354]: As is, it translates to Value, if English version is Get Value, this should be Obtener Valor

File: esp/src/src/nls/es/hpcc.ts [hunk 2]
```diff
@@ -338,8 +343,15 @@ export = {
     FromTime: "Tiempo de Comenzar",
     FullName: "Nombre Entero",
     Generate: "Generar",
+    GetDFSCSV: "CSV de DFS",
+    GetDFSMap: "Mapa DFS",
+    GetDFSParents: "Padres de DFS",
+    GetLogicalFile: "Archivo lógico",
+    GetLogicalFilePart: "Parte del archivo lógico",
     GetPart: "Obtener parte",
+    GetProtectedList: "Lista Protegida",
```
Comment (rpastrana) [inline esp/src/src/nls/es/hpcc.ts:352]: As is, it translates to Protected List, if English version is Get Protected List, this should be Obtener Lista Protegida

File: esp/src/src/nls/es/hpcc.ts [hunk 3]
```diff
@@ -338,8 +343,15 @@ export = {
     FromTime: "Tiempo de Comenzar",
     FullName: "Nombre Entero",
     Generate: "Generar",
+    GetDFSCSV: "CSV de DFS",
```
Comment (rpastrana) [inline esp/src/src/nls/es/hpcc.ts:346]: L346-350 seen to be missing "Obtener"

File: esp/src/src/nls/es/hpcc.ts [hunk 4]
```diff
@@ -621,6 +639,7 @@ export = {
     PleaseSelectAUserOrGroup: "Por favor escoja un usario o grupo y nombre de archivo",
     PleaseSelectAUserToAdd: "Por favor escoja el usario para agregar",
     Plugins: "Complementos",
+    Pods: "Vainas",
```
Comment (rpastrana) [inline esp/src/src/nls/es/hpcc.ts:642]: I think we're better off leaving this as pods, vaina is a negative word we should avoid. Also, docker seems to keep the term pod in their ES doc
https://kubernetes.io/es/docs/concepts/workloads/pods/

**Review by rpastrana — CHANGES_REQUESTED** [review]
@kunalaswani a few comments

File: esp/src/src/nls/es/hpcc.ts (hunk 3 above)
Comment (kunalaswani) [reply esp/src/src/nls/es/hpcc.ts:346]: The actual phrases for the get calls in the english file do not include get as it was not required, but the tag for it retained the get. The English version does not include Get, in the tab names on ECL Watch for reference.

Comment (rpastrana) [reply esp/src/src/nls/es/hpcc.ts:346]: fair enough, that's why I keep calling for these translation to provide the actual English text for context. 
With that said, the phrases on their own don't make much sense, but do not seem to be incorrect.

Comment (kunalaswani) [reply esp/src/src/nls/es/hpcc.ts:346]: Retained translations as is without including get to conform with the english version.

**Review by rpastrana — APPROVED** [review]
@kunalaswani good to go

**Pull request merged**